File: AnimaDemo/app/src/main/java/com/cyq/animademo/MainActivity.java
package com.cyq.animademo;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

/**
 * @author chenyangqi
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void next(View view) {
        startActivity(new Intent(this, SecondActivity.class));
    }
}

File: Rxjava/app/src/main/java/com/chenyangqi/rxjava/MainActivity.kt
package com.chenyangqi.rxjava

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.chenyangqi.rxjava.创建型操作符.CreateTypeActivity
import com.chenyangqi.rxjava.变换型操作符.TransfActivity
import com.chenyangqi.rxjava.条件型操作符.ConditionActivity
import com.chenyangqi.rxjava.过滤型操作符.FilterActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnCreateType.setOnClickListener {
            startActivity(Intent(this, CreateTypeActivity::class.java))
        }
        btnTransfType.setOnClickListener {
            startActivity(Intent(this, TransfActivity::class.java))
        }
        btnFilterType.setOnClickListener {
            startActivity(Intent(this, FilterActivity::class.java))
        }
        btnConditionType.setOnClickListener {
            startActivity(Intent(this, ConditionActivity::class.java))

        }
    }
}